.github/workflows/publish_cdn.yml

```diff
@@ -94,34 +92,20 @@ jobs:
 
       - name: Upload to blob storage
         id: storage_upload
+        uses: azure/CLI@b0e31ae20280d899279f14c36e877b4c6916e2d3
+        with:
+          inlineScript: |
+            az storage copy --account-name iopstcdnassets --destination-container \$web --source './*' --exclude-pattern "$EXCLUDED_PATH" --recursive
```

Comment:
What do you think to add destination-container, account-name and other parameters as env var?

Reply:
fixed